tools/pepsi/src/aliveness.rs

```diff
@@ -58,93 +58,137 @@ pub async fn aliveness(arguments: Arguments) -> Result<(), Error> {
     Ok(())
 }
 
-fn print_summary(states: &AlivenessList) {
-    const SPACING: usize = 3;
-    const CHARGE_OK_THRESHOLD: f32 = 0.95;
-    const CHARGING_ICON: &str = "󱐋";
-    const DISCHARGING_ICON: &str = "󰁽";
-    const OS_ICON: &str = "󱑞";
-    const ALL_OK_ICON: &str = "✔";
-    const UNKNOWN_CHARGE_ICON: &str = "󰁽?";
-    const NETWORK_ICON: &str = "󰖩 ";
-    const TEMPERATURE_ICON: &str = "";
+struct SummaryElements {
+    output: String,
+}
 
-    for (ip, state) in states.iter() {
-        let id = match ip {
-            IpAddr::V4(ip) => ip.octets()[3],
-            IpAddr::V6(ip) => ip.octets()[15],
-        };
+const SPACING: usize = 3;
+const BATTERY_CHARGE_FULL: f32 = 0.95;
+const BATTERY_CHARGE_WARN: f32 = 0.7;
+const CHARGING_ICON: &str = "󱐋";
+const DISCHARGING_ICON: &str = "󰁽";
+const OS_ICON: &str = "󱑞";
+const ALL_OK_ICON: &str = "✔";
+const UNKNOWN_CHARGE_ICON: &str = "󰁽?";
+const NETWORK_ICON: &str = "󰖩 ";
+const TEMPERATURE_ICON: &str = "";
+const TEMPERATURE_WARN_THRESHOLD: f32 = 40.0;
+const TEMPERATURE_ERROR_THRESHOLD: f32 = 60.0;
```

Comment:
Please use the values the ears use:
```            
const TEMPERATURE_LEVEL_ONE: f32 = 76.0;
const TEMPERATURE_LEVEL_TWO: f32 = 80.0;
const TEMPERATURE_LEVEL_THREE: f32 = 90.0;
```

Reply:
I added `45` for warning as this is as soon as the robot was used. and `80` for error level as this robot should first be cooled down before using it.